src/core_plugins/kibana/public/discover/embeddable/search_embeddable.js

```diff
@@ -77,11 +85,23 @@ export class SearchEmbeddable extends Embeddable {
   initializeSearchScope() {
     this.searchScope = this.$rootScope.$new();
 
-    this.pushContainerStateParamsToScope();
-
     this.searchScope.description = this.savedSearch.description;
     this.searchScope.searchSource = this.savedSearch.searchSource;
 
+    const timeRangeSearchSource = this.searchScope.searchSource.new();
+    timeRangeSearchSource.filter(() => {
+      return getTime(this.searchScope.searchSource.index(), this.timeRange);
```

Comment:
I think we might be safer using `.get('index')` instead, since this also looks up the parents (#19722). I think it's actually not needed here, since that searchSource should always be the one that has the index attached, but better safe than sorry :)